Controllers/BookingController.cs

```diff
@@ -28,4 +28,11 @@ public async Task<IActionResult> GetBookings(string userId)
         var bookings = await _bookingService.GetMyBookings(Guid.Parse(userId));
         return Ok(bookings);
     }
+
+    [HttpPut("completeBooking/{bookingId}")]
```

Comment:
You can refactor this to use conventional restful api routing.
example:
`[HttpPut("bookings/{bookingId}")]`